bin/spring

```diff
@@ -1,18 +1,15 @@
 #!/usr/bin/env ruby
 
-# This file loads spring without using Bundler, in order to be fast
-# It gets overwritten when you run the `spring binstub` command
+# This file loads spring without using Bundler, in order to be fast.
+# It gets overwritten when you run the `spring binstub` command.
```

Comment:
Just wondering if this file itself was modified as we see here by the `spring binstub` command.

Reply:
Yes that came from came from running `spring binstub --all` after updating spring.